sigs.yml

```diff
@@ -174,6 +174,24 @@
         github: danielgblanco
     roadmapProjectIDs:
       - 79
+  - name: 'Semantic Conventions: RPC'
+    meeting: Every Wednesday at 05:00 PT
+    notes:
+      type: gDoc
+      value: 1zTi_Z4WBisytPnXTpwtnGOK-4Gv5oSTuMHvIQ8--5hg
```

Comment:
it points to DB stabilization google doc, I don't mind reusing it, just wanted to make sure it's intentional

Reply:
nope, fixed!